src/cascade/input_data/db/bundle.py

```diff
@@ -257,7 +257,7 @@ def _normalize_measures(data):
     """Transform measure_ids into canonical measure names
     """
     data = data.copy()
-    data["measure"] = data.measure_id.apply(MEASURES.get)
+    data["measure"] = data.measure_id.apply(lambda k: MEASURES[k])
```

Comment:
Why go for the lambda? It's a stylistic choice of some sort.

Reply:
Ah. That works. Could also use ``MEASURES.__getitem__``. Python is against lambdas. I need to either get on board with that or get on board against it.